cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/PythonLanguage.kt

```diff
@@ -42,6 +42,7 @@ class PythonLanguage : Language<PythonLanguageFrontend>(), HasShortCircuitOperat
     override val conjunctiveOperators = listOf("and")
     override val disjunctiveOperators = listOf("or")
 
+    /** See [Documentation](https://docs.python.org/3/library/stdtypes.html#). */
     @Transient
     override val simpleTypes =
         mapOf(
```

Comment:
Should we make `bool` also a `BooleanType`? The documentation states that it's an integer type but for consistency reasons (and iff the bitwidth is 1), we should use the new type here as well.

Reply:
Would be fine by me. 
Any opinion on this, @maximiliankaul ?

Reply:
Actually, I convinced him to model it like this because I didn't want a specific `BooleanType` and referred to the documentation 😄

Reply:
Yeah, although the type "behaves" like a real boolean (unlike C, where it is really only a typedef to int usually for example). 

The documentation you refer to is probably the C API (which I also found): https://docs.python.org/3/c-api/bool.html, which specifies that its implemented as an integer of 0 / 1 internally. But this is probably due to the fact that C has no boolean to begin with, so it needs to be implemented as an int internally.

By vote is on `BooleanType`.